### PR TITLE
Force update repository manager to launch postfork.

### PR DESCRIPTION
**Pull request by dannon**

### CODE REVIEW

File: lib/galaxy/webapps/galaxy/buildapp.py [hunk 1]
```diff
@@ -128,6 +129,9 @@ def paste_app_factory( global_conf, **kwargs ):
 
     register_postfork_function(postfork_setup)
 
+    for th in threading.enumerate():
+        if th.is_alive():
+            log.debug("Prior to webapp return, Galaxy thread %s is alive.", th)
```
Comment (natefoo) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: This is useful but probably only in the uWSGI context. Can you wrap it in a check of `process_is_uwsgi`?

Comment (dannon) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: Hrmm.  I figured it'd be good to have a single picture of running threads in either context, but I'm fine with wrapping it too.

Comment (dannon) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: Actually, I think I found a bug in process_is_uwsgi, hang on.

Comment (natefoo) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: Somehow my brain translated "prior to webapp return" to "prior to fork". Disregard my comment...

Comment (dannon) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: @natefoo Ok, cool, won't wrap it.  I did find a bug though.  uwsgidecorators was _not_ in my environment but uwsgi was.  The try/except was masking this error and stating that process_is_uwsgi was false, even though it was true.  The fix in efbd8f2 resolves that, but I'm trying to figure out why the decorator package is (now?) separate and not found.

Comment (natefoo) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: Maybe a `sys.path` issue? The `uwsgi` importable is part of the uWSGI "interpreter" itself, `uwsgidecorators` is an external module installed into `site-packages`.

Comment (dannon) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: Yeah, that's what I figured out.  The problem was conflicting environments in that I was using a system uwsgi but the .venv from Galaxy, so it wasn't being found.

Comment (dannon) [inline lib/galaxy/webapps/galaxy/buildapp.py:134]: @natefoo Added a simple warning, I think this is good to go now.

**Pull request merged**